src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_LINE_NAME = "supply";
+    private static final String BUY_LINE_NAME = "buy";
+    private static final String RESULT_LINE_NAME = "result";
+    private static final String DELIMITER = ",";
+    private int supplyTotal;
+    private int buyTotal;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        supplyTotal = 0;
+        buyTotal = 0;
+        readFile(fromFileName);
+        writeToFile(createReport(), toFileName);
```

Comment:
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```
your program should look like this